front/app/modules/commercial/customizable_homepage_banner/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ModuleConfiguration } from 'utils/moduleUtils';
+import LayoutSetting from './admin/LayoutSetting';
+import TwoColumnLayout from './citizen/TwoColumnLayout';
+import TwoRowLayout from './citizen/TwoRowLayout';
+
+const configuration: ModuleConfiguration = {
+  outlets: {
+    'app.containers.Admin.settings.customize.headerSectionStart': (props) => (
+      <LayoutSetting {...props} />
```

Comment:
What if the feature is disabled in the database? It seems this component will still be loaded?
Shouldn't we use similar pattern?
```tsx
      <FeatureFlag name="customizable_homepage_banner">
        <LayoutSetting {...props} />
      </FeatureFlag>
```